moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -572,35 +605,22 @@ const ProjectFundingTable = () => {
       field: "fund",
       width: 200,
       editable: true,
-      renderCell: ({ row }) =>
-        !!row.fund?.fund_name ? (
-          <>
-            <Typography>{row.fund?.fund_id} |</Typography>
-            <Typography>{row.fund?.fund_name}</Typography>
-          </>
-        ) : (
-          ""
-        ),
+      valueFormatter: (value) =>
```

Comment:
valueFormatter instead of renderCell because renderCell creates a whole component whereas valueFormatter and valueGetter only change the values. helps with sorting too.